templates/python-start-scrape-single-page/src/__main__.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
```

Comment:
Do we really need the logger here? The default console log should be enough for this use case. I don't think we need to use the Apify one

Reply:
The SDK uses the logger internally, if this is not there, its log will be missing or ugly.

Plus I expect that people will want to use this template to create more complicated actors too, it would be weird if they don't have the logger setup. 

And this should be just an internal file that the users shouldn't need to touch anyway, that's why the logger complexity is hidden here instead of the main `main.py` file.